modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala

```diff
@@ -69,9 +67,7 @@ final case class DagRoutes[F[_]: Async](dagService: DAGService[F], mkDagCell: L0
     case req @ POST -> Root / "l1-output" =>
       req
         .as[Signed[DAGBlock]]
-        .map(L0CellInput.HandleDAGL1(_))
-        .map(mkDagCell)
-        .flatMap(_.run())
```

Comment:
why it can't be like that? service doesn't add any additional logic in that case and the real logic is handled by the dag cell anyway

Reply:
Reverted.